config/src/color.rs

```diff
@@ -82,6 +106,9 @@ impl From<Palette> for wezterm_term::color::ColorPalette {
                 p.colors.0[idx + 8] = *col;
             }
         }
+        for (idx, col) in cfg.indexed.iter() {
```

Comment:
`&cfg.indexed` is slightly nicer looking than explicitly calling `cfg.indexed.iter()`.

Also, using `&` on the binding part before the `in` allows removing the explicit `*` to deref inside the loop.

```suggestion
        for (&idx, &col) in &cfg.indexed {
```